packages/web-components/src/components/ic-classification-banner/ic-classification-banner.tsx

```diff
@@ -37,7 +37,10 @@ export class ClassificationBanner {
   @Prop() upTo?: boolean = false;
 
   render() {
-    const { classification, inline, country, upTo, additionalSelectors } = this;
+    const { classification, inline, upTo } = this;
+    let { country, additionalSelectors } = this;
+    if(!country) country = "uk";
```

Comment:
This sets the default country to "UK", which is not behaviour we want to introduce. Based on #2177, we need to be checking for the string "undefined" and handle that correctly